docs/content/v2.14/reference/configuration/yb-tserver.md

```diff
@@ -388,6 +388,12 @@ Interval at which the tablet manager tries to cleanup split tablets that are no
 
 Default: `60`
 
+##### --enable_automatic_tablet_splitting
+
+Enables YugabyteDB to [automatically split tablets](../../../architecture/docdb-sharding/tablet-splitting/#automatic-tablet-splitting), based on the specified tablet threshold sizes configured below.
+
+Default: `false`
+
```

Comment:
The flag's summary should be extended with the following text at the end:
```

{{< note title="Important" >}}

This value must match on all `yb-master` and `yb-tserver` configurations of a YugabyteDB cluster.

{{< /note >}}
```

This should be applied for every `yb-master.md` and `yb-tserver.md` for 2.14, 2.16, stable and preview versions.